app/partial-head.html

```diff
@@ -0,0 +1,5 @@
+<meta charset="utf-8" />
+<meta name="viewport" content="<%= it?.viewport ?? "width=device-width, initial-scale=1, user-scalable=no" %>" />
```

Comment:
hard to spot because of HTML formatting, but this uses the supplied `viewport` param, or a default if it isn't supplied.

I'd argue that we should actually remove `user-scalable=no`, but that's a conversation for another time.